documentation/Creating-your-templates.md

```diff
@@ -62,30 +62,13 @@ strings:
-
-| Node | Description |
-| ---- | ----------- |
-| `{% set my_capture_variable %}`<br>`{% endset %}` | node pair allowing you to capture the rendering of nodes inside it, so you can reuse it later by reusing `{{ my_capture_variable }}`.<br>This is useful if you intend to reuse the same content at multiple places — like the `sceneID` is for the `storyboards-default.stencil` template for example. |
-| `{% macro my_block a b c %}`<br>`{{a}} - {{b}} - {{c}}`<br>`{% endmacro %}` | node pair allowing you to define a reusable block of nodes inside (like a function), so you can reuse it later.<br>In contrast to the `set` tag, the capturing for `macro` is lazy, it will only be processed when actually called. When processed, the block will provide all the defined parameters as variables with their corresponding names.<br>Similar to the `for` tag, this tag provides it's own context scope when called, so variables inside of it are not available outside of it. |
-| `{% call my_block a b c %}` | node allowing you to call a previously defined block. The number of arguments must match the definition of the block. Note that you can call a block from inside the block itself, allowing a form of recursion if need be. |
-
-| Filter | Description |
-| ------ | ----------- |
-| `swiftIdentifier` | Transforms a `String` into a valid Swift identifier, replacing every invalid character to use for a Swift Identifier by a `_`.<br>This is very useful to use an arbitrary string (like a Storyboard ID) as a `case` or `enum` name. |
-| `join` | Operates on Arrays of Strings; joins elements with `, `. |
-| `lowerFirstWord` | Lowercases the first word of a camelCase text; for example transforms `FooBar` to `fooBar` and `URLChooserController` to `urlChooserController`. |
-| `snakeToCamelCase` | Transforms snake_case (using an underscore to separate words, like `foo_bar_baz`) to CamelCase (`FooBarBaz`). |
-| `titlecase` | Uppercases the first letter of a string. This is quite similar to the `capitalize` filter bundled with Stencil, but doesn't force-lowercase the other letters, so while `fooBarBaz|capitalize` will return `Foobarbaz`, `fooBarBaz|titlecase` with return `FooBarBaz`. |
-| `escapeReservedKeywords` | Checks if the string is one of the reserved keywords and if so, escapes it using backticks. While given `foo_bar` it would return `foo_bar`, given `static`, it would return `\`static\``. |
-
-See also [the `GenumKit.GenumTemplate` source code](https://github.com/AliSoftware/SwiftGen/blob/master/GenumKit/Stencil/GenumTemplate.swift#L35-L42) and [the files in `GenumKit/Stencil`](https://github.com/AliSoftware/SwiftGen/tree/master/GenumKit/Stencil).
+Additionally to the [tags and filters](https://stencil.fuller.li/en/latest/builtins.html) provided by Stencil, SwiftGen provides some additional ones, documented in the [StencilSwiftKit framework](https://github.com/SwiftGen/StencilSwiftKit).
```

Comment:
Btw the code comments there are out of date and don't match the actual structure of the generated contexts 😉